src/Tribe/Cache.php

```diff
@@ -51,13 +55,14 @@ public function set( $id, $value, $expiration = 0, $expiration_trigger = '' ) {
 		 * Filters the expiration for cache objects to provide the ability
 		 * to make non-persistent objects be treated as persistent.
 		 *
-		 * @param int          $expiration         Cache expiration time.
+		 * @since 4.8
+		 *
 		 * @param string       $id                 Cache ID.
 		 * @param mixed        $value              Cache value.
 		 * @param string|array $expiration_trigger Action that triggers automatic expiration.
 		 * @param string       $key                Unique cache key based on Cache ID and expiration trigger last run time.
 		 *
-		 * @since 4.8
+		 * @param int          $expiration         Cache expiration time.
```

Comment:
Shouldn't this be the first param listed and without a blank line?

Reply:
Yes my bad.